packages/expect/src/matchers.js

```diff
@@ -213,7 +214,7 @@ const matchers: MatchersObject = {
         isNot: this.isNot,
       }) +
       '\n\n' +
-      `Expected: ${printExpected(expected)}\n` +
+      `Expected value to be less than or equal: ${printExpected(expected)}\n` +
```

Comment:
These messages will need to be broken out into cases with different `.not` handling

Check out [here](https://github.com/facebook/jest/blob/0f24b4439f0c645aeb57accb780b8ab98b1cd512/packages/expect/src/matchers.js#L371-L392) for how we do this elsewhere